Makefile

```diff
@@ -10,12 +10,25 @@ install: init-env ## Install dependencies using composer
 	docker-compose run --rm symfony bash -ci 'composer update'
 	docker-compose run --rm symfony bash -ci 'composer install'
 
+
 start: ## Start containers in dev environment
 	docker-compose up --force-recreate -d
 
+
 stop: ## Stop containers in dev environment
 	docker-compose down
 
+install-prod: init-env ## Install dependencies using composer in prod
+	docker-compose -f docker-compose.yml -f docker-compose.prod.yml build
```

Comment:
You can add variable on top of the file like  `DOCKER_COMPOSE_DEV =` and `DOCKER_COMPOSE_PROD =`